### PR TITLE
Fix mask type of nusence dataset

### PR DESCRIPTION
**Pull request by cxiang26**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily get feedback. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation

I got a type error when I convert nuscenes dataset, the error is caused by the mask type

## Modification

Please briefly describe what modification is made in this PR.

## BC-breaking (Optional)

Does the modification introduce changes that break the back-compatibility of the downstream repos?
If so, please describe how it breaks the compatibility and how the downstream projects should modify their code to keep compatibility with this PR.

## Use cases (Optional)

If this PR introduces a new feature, it is better to list some use cases here, and update the documentation.

## Checklist

1. Pre-commit or other linting tools are used to fix the potential lint issues.
2. The modification is covered by complete unit tests. If not, please add more unit test to ensure the correctness.
3. If the modification has potential influence on downstream projects, this PR should be tested with downstream projects.
4. The documentation has been modified accordingly, like docstring or example tutorials.

### CODE REVIEW

File: mmdet3d/datasets/nuscenes_dataset.py [hunk 1]
```diff
@@ -99,7 +99,7 @@ def _filter_with_mask(self, ann_info: dict) -> dict:
         """
         filtered_annotations = {}
         if self.use_valid_flag:
-            filter_mask = ann_info['bbox_3d_isvalid']
+            filter_mask = ann_info['bbox_3d_isvalid'] > 0
```
Comment (ZCMax) [inline mmdet3d/datasets/nuscenes_dataset.py:102]: `bbox_3d_isvalid` seems store the bool value?

**Comment by ZCMax** [comment]
thanks for your PR! Please sign the CLA first~

File: mmdet3d/datasets/nuscenes_dataset.py (hunk 1 above)
Comment (cxiang26) [reply mmdet3d/datasets/nuscenes_dataset.py:102]: v1.0-mini of nusences is float

Comment (cxiang26) [reply mmdet3d/datasets/nuscenes_dataset.py:102]: I just test the v1.0-mini of nusence limited by my device, the bbox_3d_isvalid type is float

Comment (ZCMax) [reply mmdet3d/datasets/nuscenes_dataset.py:102]: A little strange here, we need to check it again

Comment (Xiangxu-0103) [reply mmdet3d/datasets/nuscenes_dataset.py:102]: I trained `nuscenes dataset` both for lidar detection and monocular detection, and everything seems to be OK. Do you use the latest code to update infos and train model?

Comment (cxiang26) [reply mmdet3d/datasets/nuscenes_dataset.py:102]: I got the error when I call ./tools/create_data.py with version=v1.0-mini rather than doing detection.

Comment (Xiangxu-0103) [reply mmdet3d/datasets/nuscenes_dataset.py:102]: I re-generate the nuscenes info with  `v1.0-mini` version just now, and everything is OK.

Comment (cxiang26) [reply mmdet3d/datasets/nuscenes_dataset.py:102]: maybe I need to upgrade my raw dataset, thanks very much

Comment (cxiang26) [reply mmdet3d/datasets/nuscenes_dataset.py:102]: I will download newest version and try it again

**Pull request closed without merging**